### PR TITLE
Adding issue labeller and greeting github action

### LINKED ISSUE #114: Add issue labeller and greeting github action
Add issue labeller action to add relevant labels to the issues once created. 

Add GitHub action to greet a first-time contributor for an issue or a PR.

### PR DESCRIPTION
**Pull request by bislara**
Added the issue labeller and greeting Github action. 

- **What does this PR do?**
Labels will be generated based on the content which is written in the issue description. like if its enhancement type, then labeller will add enhancement label, similarly for bug and other.
We can mention the keywords for a particular label in the `issue_labeller.yml` file inside the .github folder.

The greeting action will add a comment on the issue and PR for the first time contributors.
The message present in the greeting action (like congrats 🎉 on creating your first PR, etc) can cheer someone that his/her contribution is being acknowledged and encourage them to contribute more. 

- **Screenshots and/or Live Demo**

For Greetings action
image

For labeller action
image

## Fixed #114

### CODE REVIEW

File: .github/issues_labeler.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+APIs:
+  - "(api|APIs|apis|API|APIS)"
+BASH:
+  - "(Bash|bash|BASH)"
+javascript:
+  - "(JavaScript|JAVASCRIPT|Javascript|javascript|JS|js|Js)"
+python:
```
Comment (prateek3255) [inline .github/issues_labeler.yml:7]: We don't thave anything related to python in this repo, same goes for Bash and APIs

Comment (bislara) [reply .github/issues_labeler.yml:7]: Well, that's kind of optional. I can remove it if you want.

**Comment by bislara** [comment]
@prateek3255  Requested changes done

**Comment by prateek3255** [comment]
@all-contributors please add bislara for tools

**Pull request merged**